### ISSUE TITLE
Could not decorate Gitlab merge request. 'sonar.pullrequest.gitlab.instanceUrl' has not been set in scanner properties

### ISSUE THREAD

**Issue by enginates** [mc1arke/sonarqube-community-branch-plugin#201]
**Describe the bug**
I am getting this when I use those settings for VS C# solution.  i set **instanceUrl** but keep getting that error.

```
SonarScanner.MSBuild.exe begin /k:"testgitlab"  /d:sonar.host.url="http://localhost:9000" /d:sonar.login="xxxxxxxxxxxxxxxx" /d:sonar.pullrequest.key=1 /d:sonar.pullrequest.branch=test1 /d:sonar.pullrequest.base=master /d:sonar.pullrequest.gitlab.**instanceUrl**="https://gitlab.xxxxxx.com/" /d:sonar.pullrequest.gitlab.projectId=19 /d:sonar.pullrequest.gitlab.projectUrl="https://gitlab.xxxxxx.com/testuser/gitlabtest"

MsBuild.exe /t:Rebuild

SonarScanner.MSBuild.exe end /d:sonar.login="xxxxxxxxxxxxxxxx"
```

```
2020.06.28 23:18:35 ERROR ce[AXL-FQeqAN6gvW1Hfvud][o.s.c.t.p.a.p.PostProjectAnalysisTasksExecutor] Execution of task class com.github.mc1arke.sonarqube.plugin.ce.pullrequest.PullRequestPostAnalysisTask failed
java.lang.IllegalStateException: **Could not decorate Gitlab merge request. 'sonar.pullrequest.gitlab.instanceUrl' has not been set in scanner properties**
	at com.github.mc1arke.sonarqube.plugin.ce.pullrequest.gitlab.GitlabServerPullRequestDecorator.lambda$decorateQualityGateStatus$1(GitlabServerPullRequestDecorator.java:108)
	at java.base/java.util.Optional.orElseThrow(Optional.java:401)
	at com.github.mc1arke.sonarqube.plugin.ce.pullrequest.gitlab.GitlabServerPullRequestDecorator.decorateQualityGateStatus(GitlabServerPullRequestDecorator.java:108)
	at com.github.mc1arke.sonarqube.plugin.ce.pullrequest.PullRequestPostAnalysisTask.finished(PullRequestPostAnalysisTask.java:160)
	at org.sonar.ce.task.projectanalysis.api.posttask.PostProjectAnalysisTasksExecutor.executeTask(PostProjectAnalysisTasksExecutor.java:118)
	at org.sonar.ce.task.projectanalysis.api.posttask.PostProjectAnalysisTasksExecutor.finished(PostProjectAnalysisTasksExecutor.java:109)
	at org.sonar.ce.task.step.ComputationStepExecutor.executeListener(ComputationStepExecutor.java:91)
	at org.sonar.ce.task.step.ComputationStepExecutor.execute(ComputationStepExecutor.java:63)
	at org.sonar.ce.task.projectanalysis.taskprocessor.ReportTaskProcessor.process(ReportTaskProcessor.java:81)
	at org.sonar.ce.taskprocessor.CeWorkerImpl$ExecuteTask.executeTask(CeWorkerImpl.java:209)
	at org.sonar.ce.taskprocessor.CeWorkerImpl$ExecuteTask.run(CeWorkerImpl.java:191)
	at org.sonar.ce.taskprocessor.CeWorkerImpl.findAndProcessTask(CeWorkerImpl.java:158)
	at org.sonar.ce.taskprocessor.CeWorkerImpl$TrackRunningState.get(CeWorkerImpl.java:133)
	at org.sonar.ce.taskprocessor.CeWorkerImpl.call(CeWorkerImpl.java:85)
	at org.sonar.ce.taskprocessor.CeWorkerImpl.call(CeWorkerImpl.java:53)
	at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:125)
	at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:69)
	at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:78)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
2020.06.28 23:18:35 INFO  ce[AXL-FQeqAN6gvW1Hfvud][o.s.c.t.p.a.p.PostProjectAnalysisTasksExecutor] Pull Request Decoration | status=FAILED | time=46ms
```

**To Reproduce**
```
SonarScanner.MSBuild.exe begin /k:"testgitlab"  /d:sonar.host.url="http://localhost:9000" /d:sonar.login="xxxxxxxxxxxxxxxx" /d:sonar.pullrequest.key=1 /d:sonar.pullrequest.branch=test1 /d:sonar.pullrequest.base=master /d:sonar.pullrequest.gitlab.**instanceUrl**="https://gitlab.xxxxxx.com/" /d:sonar.pullrequest.gitlab.projectId=19 /d:sonar.pullrequest.gitlab.projectUrl="https://gitlab.xxxxxx.com/testuser/gitlabtest"

MsBuild.exe /t:Rebuild

SonarScanner.MSBuild.exe end /d:sonar.login="xxxxxxxxxxxxxxxx"
``

**Software Versions**
 - SonarQube Version: 8.3.1
 - Plugin Version: sq-8_2-support

**Comment by MukeshAtGit** [comment]
Same isssue with me here are the logs(cs.log):

`java.lang.IllegalStateException: Could not decorate Gitlab merge request. 'sonar.pullrequest.gitlab.instanceUrl' has not been set in scanner properties
	at com.github.mc1arke.sonarqube.plugin.ce.pullrequest.gitlab.GitlabServerPullRequestDecorator.lambda$decorateQualityGateStatus$1(GitlabServerPullRequestDecorator.java:105)
`

**Comment by MukeshAtGit** [comment]
Hi all this work for me:
https://github.com/mc1arke/sonarqube-community-branch-plugin/issues/112#issuecomment-601863331

**Comment by mc1arke** [comment]
Released in 1.5.0 of the plugin
